### PR TITLE
Added Bootstrap Custom Forms including switches

### PR DESCRIPTION
**Pull request by snickbit**
[https://getbootstrap.com/docs/4.3/components/forms/#custom-forms](url)

Added custom form styles and formats from Bootstrap 4.3 and made them the default with the option to turn them off with `Form::element()->custom(false);`

Included Radio, Checkbox, Switch, Select, Range, and File type inputs. 

I tried to make as little changes as possible and maintain the same code style.

### CODE REVIEW

File: src/FormBuilder.php [hunk 1]
```diff
@@ -429,6 +446,66 @@ private function buildHtmlAttrs(array $attributes, $appendAttrs = true): string
...
+		    		$class = 'custom-file';
+			    } elseif($input) {
+		    		$class = 'custom-file-input';
+			    } else{
+		    	    $class = 'custom-file-label';
+			    }
+			    break;
+		    default:
+		    	// do nothing
+			    dd('type', $type);
```
Comment (netojose) [inline src/FormBuilder.php]: This dd() debug function should be removed

Comment (snickbit) [reply src/FormBuilder.php]: How did I miss that? It's done.

**Comment by arezonavabiyan** [comment]
[هاست لینوکس](https://www.web3z.com)
[وب](https://www.web3z.com)

**Comment by chellmann** [comment]
Hey guys, i would really appreciate when this could merged.

**Comment by dangelion** [comment]
This would be super super useful. Any news?

**Comment by JamesPoel** [comment]
Any news on this? I am keen to add a custom file field to my application through this package.

**Comment by JamesPoel** [comment]
Any news?

**Comment by snickbit** [comment]
Sorry everyone, I requested a review after fixing the issue. Nothing else I can do.

**Comment by netojose** [comment]
@snickbit and @JamesPoel , I will be more active from now. And check all PR's and issues

**Pull request merged**